zilliqa/src/precompiles/pop_verify.rs

```diff
@@ -13,6 +13,7 @@ use crate::exec::PendingState;
 
 pub struct PopVerify;
 
+/// A custom Proof of Possession in which the bls public key, an evm address and a chain_id are signed over
 // keep in-sync with zilliqa/src/contracts/deposit.sol
 impl PopVerify {
     const POP_VERIFY_GAS_PRICE: u64 = 1_000_000u64; // FIXME: Gas Price?
```

Comment:
A nit. But since you're working on this, maybe you could add in the changes in #1335 which has been there for months.